frontend/src/libs/components/sidebar/sidebar.tsx

```diff
@@ -0,0 +1,71 @@
+import { useLocation } from 'react-router-dom';
+
+import home from '#assets/img/home.svg';
+import logo from '#assets/img/logo.svg';
+import { Link } from '#libs/components/components.js';
+import { AppRoute } from '#libs/enums/enums.js';
+
+import styles from './styles.module.scss';
+
+const Sidebar: React.FC = () => {
+  const iconSelected = 'icon-selected';
+  const iconStyles = 'icon-container';
+  const { pathname } = useLocation();
+
+  return (
+    <div className={styles['sidebar']}>
+      <nav className={styles['nav']}>
+        <Link to={AppRoute.ROOT}>
+          <div className={styles['image']}>
+            <img src={logo} alt="logo" />
+          </div>
+        </Link>
+      </nav>
+      <nav className={styles['nav']}>
+        <div className={styles['icons-container']}>
+          <button
+            className={`${
+              pathname === AppRoute.ROOT
```

Comment:
Let's use getValidClassNames and iconSelected as an optional while condition is true